GoogleForCreators/web-stories-wp
# Media: avoid inserting .mov files from media picker

## Summary

- Took Pascal's suggestion to update the `MediaPane` to check for the allowed mimetypes. This filters it from the selection. See ticket below for the implementation suggestions.
- However, a user could still upload a `.mov` (though it would be filtered from the library) and then insert it from the picker. So, I throw an error and show a snackbar message like the drophandler does to prevent that. 

## Relevant Technical Choices
- I would have liked to prevent the upload of `.mov` files but the file blob comes in as a "image/jpeg" type from the `wp.media` upload module, so it bypasses the `isValidType` check that is already in place. Thus the error handling on the `MediaPane` itself.



## To-do



## User-facing changes
When attempting to upload and then insert a `.mov` file the user should see the following message: 
![Screen Shot 2020-11-03 at 1 45 55 PM](https://user-images.githubusercontent.com/41136059/98038763-e9f29f80-1dda-11eb-8da0-a641f886d334.png)



## Testing Instructions
- go to the local media tab on the media pane
- click upload
- upload a `.mov` file 
- The `.mov` file should NOT show in the gallery
- Attempt to insert the media by clicking Insert into Page
- Verify the media is not inserted and the snackbar message appears informing the user of the allowed media types. 



---



Fixes #4931
